jvm/src/test/scala/HelpTest.scala

```diff
@@ -221,7 +221,7 @@ class HelpTest extends UsefulMatchers with CapturingTest {
         }
         addSubcommand(peach)
 
-        val submarine = new Subcommand("submarine") {()}
+        val submarine = new Subcommand("submarine") {}
```

Comment:
Nice catch :) This was actually required several years ago (due to our DelayedInit schenanigans), but thankfully we don't need that anymore.

Reply:
Yeah, those were just complaints by the compiler ("warning")